tb_plugin/torch_tb_profiler/run.py

```diff
@@ -82,17 +82,31 @@ def build_trace_counter_sm_efficiency(gpu_id, start_time, counter_value):
             )
             return util_json
 
+        def add_trace_counter_gpu_util(gpu_id, start_time, counter_value):
+            json_str = build_trace_counter_gpu_util(gpu_id, start_time, counter_value)
+            counter_json_list.append(json_str)
```

Comment:
pass by parameter to make it clear so that this method could be move outside of current method.

Reply:
Good. Done. Thanks.